web/atlas.json

```diff
@@ -3516,7 +3515,7 @@
 {"id": "tx28l8", "submitted_by": "Stuart98", "name": "The University of Utah", "description": "Utah's flagship public university, located in Salt Lake City, Utah.", "website": "https://www.utah.edu/", "subreddit": "/r/uofu", "center": [1978.5, 873.5], "path": [[1970.5, 870.5], [1970.5, 876.5], [1986.5, 876.5], [1986.5, 870.5], [1970.5, 870.5]]},
 {"id": "tx28jj", "submitted_by": "distanceluminosity", "name": "The Defenders", "description": "A group of small communities that created an alliance to fight against the void. Went down in the final hours of r/place fighting the void.", "website": "", "subreddit": "", "center": [1110.5, 1201.5], "path": [[1099.5, 1192.5], [1099.5, 1208.5], [1109.5, 1208.5], [1109.5, 1212.5], [1119.5, 1212.5], [1119.5, 1191.5], [1109.5, 1191.5], [1109.5, 1192.5]]},
 {"id": "tx28iv", "submitted_by": "Cat2468", "name": "Aipha Banner", "description": "The logo of the streamer Aipha", "website": "https://twitch.tv/Aipha", "subreddit": "", "center": [1193.5, 1011.5], "path": [[1182.5, 999.5], [1204.5, 999.5], [1205.5, 1023.5], [1182.5, 1023.5], [1182.5, 1023.5], [1181.5, 1023.5]]},
-{"id": "tx27xv", "submitted_by": "InsaneByte", "name": "SWAD", "description": "SWAD is an educational platform of the University of Granada originally created for teaching, its creator Antonio Cañas Vargas, professor of computer engineering and computer architecture, began to add features that students requested and the students ended up turning it into a shitpost platform loved in the ETSIIT (technical school of computer engineering and telecommunications) of Granada.nA group of 40 students came together to create the r/place logo.n", "website": "https://swad.ugr.es/es", "subreddit": "", "center": [916.5, 1320.5], "path": [[901.5, 1315.5], [901.5, 1325.5], [931.5, 1325.5], [931.5, 1315.5], [901.5, 1315.5], [901.5, 1315.5]]},
+{"id": "tx27ur", "submitted_by": "bisc0tte_", "name": "Dakota", "description": "PearlescentMoon's cat!", "website": "https://youtube.fandom.com/wiki/PearlescentMoon", "subreddit": "/r/hermitcraft", "center": [899.5, 627.5], "path": [[897.5, 624.5], [901.5, 624.5], [901.5, 630.5], [897.5, 630.5], [897.5, 624.5]]},

```

Comment:
This entry was modified by someone else, but your change brought back the old version. Remove this old version of the entry.